src/reducers/ui.js

```diff
@@ -105,6 +106,15 @@ export default (state = EMPTY_UI, action) => {
                 program: EMPTY_UI.program,
             }
         }
+        case CLEAR_UI_STAGE: {
+            return {
+                ...state,
+                program: {
+                    id: state.program.id,
+                    stage: undefined,
```

Comment:
This property is now named `stageId` and accordingly action UPDATE_UI_PROGRAM_STAGE_ID. So this action should also be renamed (and clear the value of `stageId`)

Reply:
I'm wondering if the properties of the default `program` property ought to include the `id` and `stageId` (set to undefined) to make clear the expected structure.